apps/web/components/experiment-settings/experiment-details-card.tsx

```diff
@@ -71,7 +71,18 @@ export function ExperimentDetailsCard({
                 <FormItem>
                   <FormLabel>{t("experimentSettings.name")}</FormLabel>
                   <FormControl>
-                    <Input {...field} placeholder={t("experimentSettings.name")} />
+                    <Input
+                      {...field}
+                      onBlur={(e) => {
+                        // Trim whitespace from the input value
+                        const trimmed = e.target.value.trim();
```

Comment:
I would never trim on the client side, it's even a question, should we do it on the backend, a space for better or worse is a valid input